src/test/java/org/symphonyoss/symphony/messageml/elements/form/ButtonTest.java

```diff
@@ -96,7 +97,8 @@ public void testButtonWithValidClasses() throws Exception {
     String innerText = "Class Test";
 
     for (String clazz : VALID_CLASSES) {
-      String input = "<messageML><form id=\"" + FORM_ID_ATTR + "\"><button type=\"" + type + "\" class=\"" + clazz + "\">" + innerText
+      String formId = FORM_ID_ATTR + RandomStringUtils.random(10, true, true);
+      String input = "<messageML><form id=\"" + formId + "\"><button type=\"" + type + "\" class=\"" + clazz + "\">" + innerText
```

Comment:
This scenario was using the same context object to parse two different inputs using the same form ID, so I had to modify it in order to create different ids for each parse call to avoid a false positive here.